.github/CONTRIBUTING.md

```diff
@@ -363,9 +363,12 @@ There is no strict process when it comes to merging pull requests. Pull requests
 
 ## Banned content
 Do not add any of the following in a Pull Request or risk getting the PR closed:
+* Code that interacts with, sends data to, receives data from, or other wise has anything to do with Kite, Kite.com, or any affiliate companies.
 * National Socialist Party of Germany content, National Socialist Party of Germany related content, or National Socialist Party of Germany references
+* Code, Pull Requests, or Issue Reports submitted by Kite, Kite.com, or their employees or affiliates
```

Comment:
whom are acting as agents of Kite or Kite.com or otherwise advancing the interests of Kite or Kite.com.